bazel/README.md

```diff
@@ -272,12 +272,14 @@ for how to update or override dependencies.
 
 ## Running the built Envoy binary on the host system
 
-After Envoy is built, it can be executed via CLI.
+After Envoy is built, it can be executed via CLI. 
+
+As Bazel's current ability to provide information about where it just built the artifacts is limited (as stated in [bazel's issue #8739](https://github.com/bazelbuild/bazel/issues/8739)), we can use the `bazel info bazel-genfiles` command to get the path where Envoy was built.
```

Comment:
i dont think we want to include this message - sorry if i was unclear - this was a message to you rather than a suggestion

Reply:
Got it, i'll make the changes

Reply:
What about the line:

>  we can use the `bazel info bazel-genfiles` command to get the path where Envoy was built

Do you think it's worth mentioning? Or seems unnecessary

Reply:
on balance, probably unnecessary